backend/lucene/src/main/java/org/hibernate/search/backend/lucene/types/predicate/impl/LuceneTextFieldPredicateBuilderFactory.java

```diff
@@ -61,7 +61,7 @@ public LuceneTextFieldPredicateBuilderFactory( boolean searchable,
 	public WildcardPredicateBuilder<LuceneSearchPredicateBuilder> createWildcardPredicateBuilder(
 			String absoluteFieldPath) {
 		checkSearchable( absoluteFieldPath );
-		return new LuceneTextWildcardPredicateBuilder( absoluteFieldPath );
+		return new LuceneTextWildcardPredicateBuilder( absoluteFieldPath, analyzerOrNormalizer );
```

Comment:
There's also a "prefix" syntax in the simple query string predicate. Could you please implement the fix and test it for that predicate, too?

Reply:
Sorry I missed completely the `prefix` part. Adding now...